revolv/static/project.scss

```diff
@@ -43,6 +43,439 @@
+                position: absolute;
+
+                left: 50%;
+                top: 60%;
+                -webkit-transform: translate(-50%, -50%);
+                -ms-transform: translate(-50%, -50%);
+                transform: translate(-50%, -50%);
+            }
+        }
+    }
+}
+
+.about {
+    margin-top: 200px;
+
+    .header-text {
+        p {
+            font-weight: 800;
+            color: black;
+            font-size: 1.8rem;
+        }
+    }
+    .gray-bar {
+        background-color: #999999;
+        height: 1px;
+        width: 100%;
+    }
+
+    .categories {
+        max-width: 100%;
+        height: 50px;
+
+
+        .category-text {
+            display: inline-block;
+            color: #888888;
+            margin-left: 15px;
+            margin-right: 10px;
+            height: 30px;
+
+            .category-text-inside {
+                line-height: 30px;
+                font-size: 0.8rem;
+            }
+        }
+
+        .category-holder {
+            display: inline-block;
+            border-color: #888888;
+            border-style: solid;
+            border-width: 1px;
+            width: 15%;
+            height: 30px;
+            margin-top: 10px;
+            margin-right: 5px;
+            margin-left: 5px;
+
+            .category-label {
+                color: #888888;
+                text-align: center;
+                line-height: 30px;
+            }
+        }
+    }
+
+    .about-text {
+        p {
+            font-weight: 500;
+            font-size: 1.5rem;
+        }
+        margin-bottom: 80px;
+    }
+
+}
+
+.project-updates {
+    .header-text {
+        p {
+            font-weight: 800;
+            color: black;
+            font-size: 1.8rem;
+        }
+    }
+    .gray-bar {
+        background-color: #999999;
+        height: 1px;
+        width: 100%;
+    }
+
+    .update-container {
+        margin-top: 40px;
+        width: 100%;
+        margin-bottom: 40px;
+        display: inline-block;
+        position: relative;
+
+        .date-container {
+            height: 50px;
+            width: 100%;
+            text-align: center;
+            color: #064f7e;
+            font-weight: 300;
+            line-height: .5;
+
+            .date-year {
+                font-size: 4.5rem;
+                margin-bottom: 0rem;
+                line-height: 1;
+            }
+
+            .date-month-day {
+                font-size: 1.5rem;
+                line-height: 1;
+            }
+        }
+
+        .update-text {
+            font-weight: 500;
+            font-size: 1.5rem;
+        }
+    }
+}
+
+.map {
+    height: 400px;
+    margin-top: 40px;
+    margin-bottom: 40px;
+
+    #map-canvas {
+        width: 100%;
+        height: 100%;
+    }
+
+
+    .overlay {
+        background: rgba(0, 0, 0, .5);
+        float: left;
+        height: 400px;
+        width: 40%;
+        position: absolute;
+        margin-top: -400px;
```

Comment:
I think this div should be position: absolute with height: 100%. That way you don't need the hacky negative margin